rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -108,7 +108,8 @@ def get_all_info(
         May raise:
         - RemoteError
         """
-        result, total_nfts_num = self._get_all_nft_data(addresses, ignore_cache=ignore_cache)
+        _result, total_nfts_num = self._get_all_nft_data(addresses, ignore_cache=ignore_cache)
```

Comment:
since you add the `_filter_ignored_nfts` right after `get_all_nft_data` in both places it appears why don't you just make it part of the function instead?